test/WebSites/HtmlGenerationWebSite/Startup.cs

```diff
@@ -14,9 +14,8 @@ public class Startup
         // Set up application services
         public void ConfigureServices(IServiceCollection services)
         {
-            // Add MVC services to the services container. Change default FormTagHelper.AntiForgery to false. Usually
-            // null which is interpreted as true unless element includes an action attribute.
-            services.AddMvc().InitializeTagHelper<FormTagHelper>((helper, _) => helper.Antiforgery = false);
+            // Add MVC services to the services container.
+            services.AddMvc();
```

Comment:
Undo this. The site is _intentionally_ testing e2e behaviour with the `Antiforgery` property set to `false`.